### DISCUSSION TITLE
We are the target of SPAM!!  Please be careful

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2585] [General]
We have received spam all of a sudden.
I have reported it to GitHub already.

I am posting in the thread pointing out the spam message.

We will share the info with you as soon as we hear back from GitHub.

In the meantime, please do not open the links if you have been tagged.

**Comment by PhilTU** [comment]
BarbaraPerez2968 by any chance ?

**Reply by Navid200** [reply]
That might have been one of them.  
I submitted a report containing 6 accounts and 10 posts, and this was the response.  
  
![Screenshot 2022-12-29 132752](https://user-images.githubusercontent.com/51497406/209994204-93fc393f-715c-4cc9-8819-49ed50cf6869.png)
